Add support for Linux Wayland

Fyne 1.4.2 adds priliminary support for Wayland, but fyne-cross is missing some dependencies.

```
❯ fyne-cross linux -arch arm -tags wayland
[i] Target: linux/arm
[i] Cleaning target directories...
[✓] "bin" dir cleaned: /tmp/fyne/cmd/fyne_demo/fyne-cross/bin/linux-arm
[✓] "dist" dir cleaned: /tmp/fyne/cmd/fyne_demo/fyne-cross/dist/linux-arm
[✓] "temp" dir cleaned: /tmp/fyne/cmd/fyne_demo/fyne-cross/tmp/linux-arm
[i] Checking for go.mod: /tmp/fyne/cmd/fyne_demo/go.mod
[✓] go.mod found
[i] Building binary...
# github.com/go-gl/glfw/v3.3/glfw
In file included from ./glfw/src/internal.h:190,
                 from ./glfw/src/context.c:30,
                 from /go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20200625191551-73d3c3675aa3/c_glfw.go:4:
./glfw/src/wl_platform.h:28:10: fatal error: xkbcommon/xkbcommon.h: No such file or directory
 #include <xkbcommon/xkbcommon.h>
          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[✗] exit status 2
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,7 +3,8 @@
 ## Unreleased
 - Add support for "fyne release" #3
 - Add support for creating packaged .tar.gz bundles on freebsd #6
-- Update fyne cli to v1.4.2-0.20201125075943-97ad77d2abe0 (fyne-io#1538 fyne-io#1527)
+- Add support for Linux Wayland #10
+- Update fyne cli to v1.4.2-0.20201204171445-8f33697cf611 (fyne-io#1538 fyne-io#1527)
 
 ## [0.9.0]
 - Releaseing under project namespace with previous 2.2.1 becoming 0.9.0 in fyne-io namespace

--- a/docker/CHANGELOG.md
+++ b/docker/CHANGELOG.md
@@ -5,6 +5,10 @@ All notable changes to the fyne-cross docker images will be documented in this f
 Release cycle won't follow the fyne-cross one, so the images will be tagged using the label
 year.month.day along with the latest one.
 
+# Release 20.12.05
+- Update fyne cli to v1.4.2-0.20201204171445-8f33697cf611
+- Add support for Linux Wayland #10
+
 # Release 20.11.28
 - Update fyne cli to v1.4.2-0.20201127180716-f9f91c194737 fyne-io#1609
 

--- a/docker/base/Dockerfile
+++ b/docker/base/Dockerfile
@@ -1,7 +1,7 @@
 # docker cross 1.13.15
 ARG DOCKER_CROSS_VERSION=sha256:11a04661d910f74c419623ef7880024694f9151c17578af15e86c45cdf6c8588
 # fyne stable branch
-ARG FYNE_VERSION=f9f91c1947370189c73384fb12ee799b1cf68638
+ARG FYNE_VERSION=8f33697cf6117da329412c6614e188fa55423b85
 
 # Build the fyne command utility
 FROM dockercore/golang-cross@${DOCKER_CROSS_VERSION} AS fyne
@@ -30,6 +30,8 @@ RUN apt-get update -qq \
         unzip \
         # headers needed by xorg-dev
         x11proto-dev \
+        # deps to support wayland
+        libxkbcommon-dev \
     && apt-get -qy autoremove \
     && apt-get clean \
     && rm -r /var/lib/apt/lists/*;

--- a/docker/linux-386/Dockerfile
+++ b/docker/linux-386/Dockerfile
@@ -40,5 +40,7 @@ RUN dpkg --add-architecture i386 \
         libxxf86vm-dev:i386 \
         xserver-xorg-dev:i386 \
         xtrans-dev:i386 \
+        # deps to support wayland
+        libxkbcommon-dev:i386 \
     && apt-get clean \
     && rm -r /var/lib/apt/lists/*;

--- a/docker/linux-arm/Dockerfile
+++ b/docker/linux-arm/Dockerfile
@@ -40,5 +40,7 @@ RUN dpkg --add-architecture armhf \
         libxxf86vm-dev:armhf \
         xserver-xorg-dev:armhf \
         xtrans-dev:armhf \
+        # deps to support wayland
+        libxkbcommon-dev:armhf \
     && apt-get clean \
     && rm -r /var/lib/apt/lists/*;

--- a/docker/linux-arm64/Dockerfile
+++ b/docker/linux-arm64/Dockerfile
@@ -40,5 +40,7 @@ RUN dpkg --add-architecture arm64 \
         libxxf86vm-dev:arm64 \
         xserver-xorg-dev:arm64 \
         xtrans-dev:arm64 \
+        # deps to support wayland
+        libxkbcommon-dev:arm64 \
     && apt-get clean \
     && rm -r /var/lib/apt/lists/*;

